ci_container.sh

```diff
@@ -0,0 +1,8 @@
+IMAGE_NAME=wptd-testrun-jenkins
+
+docker build -t $IMAGE_NAME .
```

Comment:
nit: `"${IMAGE_NAME}"`

Reply:
Done. Why is the bare `$IMAGE_NAME` undesirable?

Reply:
If it ever contained spaces or special characters, then the meaning of the command would change.